### ISSUE TITLE
Incompatible with .NET 5.0

### ISSUE THREAD

**Issue by marcusrunge** [reflectronic/mediacapture-wpf#1]
Your example project is not simply compatible with .NET 5.0.
Using the CsWinRT tool does not work.
Using <TargetFramework>net5.0-windows10.0.19041.0</TargetFramework> does not work.
Removing Microsoft.Windows.SDK.Contracts doest not work.
If I can find a solution, I file a PR.

**Comment by reflectronic** [comment]
Yes, the XAML Islands one will not work. The XAML APIs are not supported in CsWinRT. There is a way to hack around it and generate that yourself, but there are bugs.
